lms/static/js/views/fields.js

```diff
@@ -421,20 +461,29 @@
 
             fieldType: 'textarea',
 
-            templateSelector: '#field_textarea-tpl',
+            fieldTemplate: field_textarea_template,
+
+            events: function () {
+                var events_hash = {
+                    'click .wrapper-u-field': 'startEditing',
+                    'click .u-field-placeholder': 'startEditing',
+                    'focusout textarea': 'finishEditing',
+                    'change textarea': 'adjustTextareaHeight',
+                    'keyup textarea': 'adjustTextareaHeight',
+                    'keydown textarea': 'onKeyDown',
+                    'paste textarea': 'adjustTextareaHeight',
+                    'cut textarea': 'adjustTextareaHeight'
+                };
 
-            events: {
-                'click .wrapper-u-field': 'startEditing',
-                'click .u-field-placeholder': 'startEditing',
-                'focusout textarea': 'finishEditing',
-                'change textarea': 'adjustTextareaHeight',
-                'keyup textarea': 'adjustTextareaHeight',
-                'keydown textarea': 'onKeyDown',
-                'paste textarea': 'adjustTextareaHeight',
-                'cut textarea': 'adjustTextareaHeight'
+                if(this.bindEvents || _.isUndefined(this.bindEvents)) {
+                    return events_hash;
+                } else {
+                    return {};
+                }
             },
 
             initialize: function (options) {
+                this.bindEvents = options.bindEvents;
```

Comment:
Instead of binding/unbinding all events (some of which would be required for other things), I think it would make sense to have a variable `persistChanges` and check its value in the relevant handler.